aem-classification-maps/aem-classification-map-repo-annotations/src/main/resources/biz/netcentric/filevault/validator/maps/aem-classification-map-repo-annotations.map

```diff
@@ -609,7 +626,329 @@
 /libs/dam/templates/mediaportal,INTERNAL
 /libs/dam/widgets,PUBLIC
 /libs/experience-fragments/config/rewriter/adobe-target-offer,INTERNAL
+/libs/fd/aep,INTERNAL
+/libs/fd/aep/content,PUBLIC
+/libs/fd/af-expeditor,INTERNAL
+/libs/fd/af/authoring/clientlibs,INTERNAL
+/libs/fd/af/authoring/components/dor,FINAL
+/libs/fd/af/authoring/components/dor/dorProperties/cq:dialog/content/items/accordian/items/basic,INTERNAL
+/libs/fd/af/authoring/components/dor/dorPropertiesTabs,FINAL
+/libs/fd/af/authoring/components/granite,FINAL
+/libs/fd/af/authoring/components/granite/demarker,INTERNAL
+/libs/fd/af/authoring/components/granite/drawer,INTERNAL
+/libs/fd/af/authoring/components/granite/form/afImageStyleSelect,INTERNAL
+/libs/fd/af/authoring/components/granite/form/autocomplete,INTERNAL
+/libs/fd/af/authoring/components/granite/form/bindrefselector,FINAL

```

Comment:
Wasn't this the one you were stumbling on @royteeuwen? Seems the classification has been fixed (at least with the AEMaaCS add-on).

Reply:
Indeed, sadly enough not for aem 6.5.23 / aem 6.6.1 :(... still in InternalArea there...